test/test_ion_binary.cpp

```diff
@@ -406,7 +421,7 @@ void test_ion_binary_reader_threshold_for_int64_as_big_int(BYTE *data, size_t le
     hREADER reader;
     ION_TYPE type;
     int64_t value;
-    ION_INT big_int_expected;
+    ION_INT *big_int_expected;
```

Comment:
Moved `big_int_expected` to be heap allocated, to make releasing with `ion_int_free` possible. Otherwise freeing the internal data for an ION_INT requires implementation knowledge, which can change.